diff --git a/wscript b/wscript
index 08ffc9632..050046677 100644
--- a/wscript
+++ b/wscript
@@ -1074,25 +1074,23 @@ def build(ctx):
     ctx.manpage(8, "ntpclients/ntpsnmpd-man.txt")
 
     # Skip running unit tests on a cross compile build
-    if not ctx.env.ENABLE_CROSS:
-        # Force re-running of tests.  Same as 'waf --alltests'
-        if ctx.cmd == "check":
-            ctx.options.all_tests = True
-
-            # Print log if -v is supplied
-            if ctx.options.verbose:
-                ctx.add_post_fun(test_print_log)
-
-        # Write test log to a file
-        ctx.add_post_fun(test_write_log)
+    if ctx.env.ENABLE_CROSS:
+        pprint("YELLOW", "Unit test runner skipped on a cross-compiled build.")
+        from waflib import Options
+        Options.options.no_tests = True
+    elif ctx.cmd == "build":
+        from waflib import Options
+        Options.options.no_tests = True
+    else:
+        # Print log
+        ctx.options.all_tests = True
 
         # Print a summary at the end
         ctx.add_post_fun(waf_unit_test.summary)
         ctx.add_post_fun(waf_unit_test.set_exit_code)
-    else:
-        pprint("YELLOW", "Unit test runner skipped on a cross-compiled build.")
-        from waflib import Options
-        Options.options.no_tests = True
+
+        # Write test log to a file
+        ctx.add_post_fun(test_print_log)
 
     if ctx.cmd == "build":
         if "PYTHONPATH" in os.environ:
