diff --git a/wscript b/wscript
index 08ffc9632..8f172b6fc 100644
--- a/wscript
+++ b/wscript
@@ -1074,7 +1074,7 @@ def build(ctx):
     ctx.manpage(8, "ntpclients/ntpsnmpd-man.txt")
 
     # Skip running unit tests on a cross compile build
-    if not ctx.env.ENABLE_CROSS:
+    if not ctx.cmd == "build":
         # Force re-running of tests.  Same as 'waf --alltests'
         if ctx.cmd == "check":
             ctx.options.all_tests = True
@@ -1090,7 +1090,7 @@ def build(ctx):
         ctx.add_post_fun(waf_unit_test.summary)
         ctx.add_post_fun(waf_unit_test.set_exit_code)
     else:
-        pprint("YELLOW", "Unit test runner skipped on a cross-compiled build.")
+        pprint("YELLOW", "Unit test runner skipped in build.")
         from waflib import Options
         Options.options.no_tests = True
 
