diff --git a/tests/wscript b/tests/wscript
index 19acc5032..1621f714e 100644
--- a/tests/wscript
+++ b/tests/wscript
@@ -101,21 +101,6 @@ def build(ctx):
         "ntpd/restrict.c",
     ] + common_source
 
-    ctx.ntp_test(
-        defines=unity_config + ["TEST_NTPD=1"],
-        features="c cprogram bld_include src_include test",
-        includes=["%s/tests/unity/" % srcnode,
-                  "%s/ntpd/" % srcnode,
-                  "%s/tests/libntp/" % srcnode,
-                  "%s/tests/common/" % srcnode,
-                  ],
-        install_path=None,
-        source=ntpd_source,
-        target="test_ntpd",
-        use="ntpd_lib libntpd_obj unity ntp "
-            "M PTHREAD CRYPTO RT SOCKET NSL",
-    )
-
     testpylib.get_bld().mkdir()
 
     pypath = pylib.get_bld()
diff --git a/wscript b/wscript
index 050046677..e7ffcf27a 100644
--- a/wscript
+++ b/wscript
@@ -1020,59 +1020,18 @@ def build(ctx):
         # the build directory never happens.  This is how we foil that.
         ctx.add_pre_fun(lambda ctx: ctx.exec_command("rm -f pylib/*.py[co]"))
 
-    if ctx.env.ENABLE_DOC_USER:
-        if ctx.variant != "main":
-            ctx.recurse("docs")
-
     if ctx.variant == "host":
         ctx.recurse("ntpd")
         return
 
-    if ctx.env.REFCLOCK_GENERIC or ctx.env.REFCLOCK_TRIMBLE:
-        # required by the generic and Trimble refclocks
-        ctx.recurse("libparse")
     ctx.recurse("libntp")
-    ctx.recurse("ntpd")
-    ctx.recurse("ntpfrob")
-    ctx.recurse("ntptime")
     ctx.recurse("pylib")
-    ctx.recurse("attic")
     ctx.recurse("tests")
 
-    # Make sure the python scripts compile, but don't install them
-    ctx(
-        features="py",
-        source=python_scripts,
-        install_path=None,
-    )
-
-    scripts = ["ntpclients/ntpleapfetch"] + python_scripts
-
-    ctx(
-        features="subst",
-        source=scripts,
-        target=[x.replace('.py', '') for x in scripts],
-        chmod=Utils.O755,
-        install_path='${BINDIR}',
-    )
-
     ctx.add_post_fun(afterparty)
     if ctx.cmd == 'clean':
         afterparty(ctx)
 
-    ctx.manpage(1, "ntpclients/ntploggps-man.txt")
-    ctx.manpage(1, "ntpclients/ntpdig-man.txt")
-    ctx.manpage(1, "ntpclients/ntpmon-man.txt")
-    ctx.manpage(1, "ntpclients/ntpq-man.txt")
-    ctx.manpage(1, "ntpclients/ntpsweep-man.txt")
-    ctx.manpage(1, "ntpclients/ntptrace-man.txt")
-    ctx.manpage(1, "ntpclients/ntpviz-man.txt")
-    ctx.manpage(1, "ntpclients/ntplogtemp-man.txt")
-    ctx.manpage(8, "ntpclients/ntpkeygen-man.txt")
-    ctx.manpage(8, "ntpclients/ntpleapfetch-man.txt")
-    ctx.manpage(8, "ntpclients/ntpwait-man.txt")
-    ctx.manpage(8, "ntpclients/ntpsnmpd-man.txt")
-
     # Skip running unit tests on a cross compile build
     if ctx.env.ENABLE_CROSS:
         pprint("YELLOW", "Unit test runner skipped on a cross-compiled build.")
